src/response.js

```diff
@@ -119,7 +141,8 @@ export default class Response extends Body {
 	}
 
 	static error() {
-		const response = new Response(null, {status: 0, statusText: ''});
+		// default error status code = 400
+		const response = new Response(null, {status: 400, statusText: ''});
```

Comment:
I would argue for a default 500, as 400 normally indicates errors done by users. 

Imagine this, if a client forgets to define their status code and uses a default one, their users should be seeing a 500 as it's typically a mistake from server side.

Another note is I believe logical changes like this should be reflected in doc.